### ISSUE TITLE
Update store distance calculation

### ISSUE THREAD

**Issue by wangannie** [calblueprint/dccentralkitchen#68]
Currently, store distance 'miles away' from the user's current location is calculated by the distance between long/lat coordinates. This is pretty inaccurate and not very helpful in real use, so we should look into a way to calculate the distance by walking or driving directions.

**Comment by anniero98** [comment]
TLDR I think this is a design decision, but here are some reasons why it makes sense to use Euclidean distance for display in the store list:
- If you look at Yelp or Google Maps, when you search it’ll display the Euclidean distance. Then, when you click into an item, the detailed view /card converts the display distance to shortest time by {driving, walking} for Maps & shortest distance by whatever Yelp decided to use for Yelp. - To use driving/walking we would need to add a user setting for whether they prefer to see driving/walking (naively) or some other decision on when to display either. IMO it makes sense have to separate out that decision of using driving/walking time/distance into the actual usage of maps; currently we don’t have maps integrated in the cards (if we did, it might look how Yelp or a fitness tracker does).
- Displaying the shortest-distance by walking or driving also assumes that customers will take the route calculated by {new method} when that might not be the case (maybe they always walk the same way, so they don't ever use Google maps), so displaying that value is problematic in that sense.
Ultimately I think this is probably a design thing - deciding if we want to add a tooltip/note on how distance is calculated or change what happens when you click into a card or something idk haha
